docs/core/tools/dotnet-test.md

```diff
@@ -380,8 +400,183 @@ You can enclose expressions in parenthesis when using conditional operators (for
 
 For more information and examples on how to use selective unit test filtering, see [Running selective unit tests](../testing/selective-unit-tests.md).
 
-## See also
+#### See also
 
 - [Frameworks and Targets](../../standard/frameworks.md)
 - [.NET Runtime Identifier (RID) catalog](../rid-catalog.md)
 - [Passing runsettings arguments through commandline](https://github.com/microsoft/vstest/blob/main/docs/RunSettingsArguments.md)
+
+### [dotnet test with MTP](#tab/dotnet-test-with-mtp)
+
+**This article applies to:** ✔️ .NET 10 SDK and later versions
+
+#### Synopsis
+
+```dotnetcli
+dotnet test
+    [-a|--arch <ARCHITECTURE>]
+    [-c|--configuration <CONFIGURATION>]
+    [--directory <DIRECTORY_PATH>]
+    [-mptm|--max-parallel-test-modules <NUMBER>]
+    [--no-build]
+    [--no-restore]
+    [--project <PROJECT_PATH>]
+    [--solution <SOLUTION_PATH>]
+    [-t|--list-tests]
+    [--test-modules <EXPRESSION>] [--root-directory <ROOT_PATH>]
+    [--no-ansi]
+    [--no-progress]
+    [--output <VERBOSITY_LEVEL>]
+    [<args>...]
+
+dotnet test -h|--help
+```
+
+#### Description
+
+With Microsoft Testing Platform, `dotnet test` operates faster than with VSTest. The test related arguments are no longer fixed as they are tied to the registered extensions in the test project(s). Moreover, it supports globbing filter when running tests. For more information, see [Microsoft.Testing.Platform](../testing/unit-testing-platform-intro.md).
+
+> [!WARNING]
+> `dotnet test` doesn't run in an environment where we have test projects using VSTest and Microsoft Testing Platform in the same solution, as the two platforms have different command line options and different features.
```

Comment:
Maybe just say that the 2 platforms are mutually incompatible without going in details?